### ISSUE TITLE
3.10.0a0 Yarl Compile Error

### ISSUE THREAD

**Issue by cooperlees** [aio-libs/yarl#459]
Seems there is a problem compiling with `__Pyx_PyList_Append`:

```
 gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security -fPIC -I/home/travis/virtualenv/python3.10-dev/include -I/opt/python/3.10-dev/include/python3.10 -c yarl/_quoting.c -o build/temp.linux-x86_64-3.10/yarl/_quoting.o
    yarl/_quoting.c: In function ‘__Pyx_PyList_Append’:
    yarl/_quoting.c:1245:23: error: lvalue required as left operand of assignment
             Py_SIZE(list) = len+1;
                           ^
    error: command '/usr/bin/gcc' failed with exit code 1
```

Full build: https://travis-ci.org/github/pypa/bandersnatch/jobs/694378588
Seems it might be a Cython bug since lxml is having the issues too.

**Comment by webknjaz** [comment]
Right, it should be a Cython bug so report this to their tracker or maybe try Cython from master.

**Comment by JustinTArthur** [comment]
It's fixed in the upcoming Cython 0.29.20, but it since the pypi package for yarl ships with pre-cythoized C code, this will still break (tracked in aio-libs/yarl#457)

**Comment by asvetlov** [comment]
yarl 1.5.0 will be published with gerenerated C files made by Cython 0.29.21
